.github/workflows/adminBundleSize.yml

```diff
@@ -0,0 +1,28 @@
+name: Admin bundle-size
+
+on:
+  pull_request:
+    branches:
+      - '**'
+    paths:
+      - '**/admin/src/**.js'
+      - '**/helper-plugin/lib/src/**.js'
+      - '**/translations/**.json'
+      - 'yarn.lock'
```

Comment:
should we target package.json files too? (in case the yarn.lock was not updated by someone)

Reply:
My idea behind this was, that every change in a package.json should be reflected in the lockfile (but not the other way around). I don't see a problem adding them, but I thought it is not necessary.

Reply:
I remember seeing PRs that changed a package.json without updating the lockfile but indeed in theory the lockfile is enough and we should catch that earlier